Makefile

```diff
@@ -381,34 +389,69 @@ ifeq ($(EXPORT_RESULT), true)
 	gocov convert profile.cov | gocov-xml > coverage.xml
 endif
 
-.PHONY: test-clusterctl-create
+.PHONY: test-cluster-create
 test-clusterctl-create: ## Run the tests using clusterctl
```

Comment:
```suggestion
.PHONY: test-cluster-create
test-cluster-create: ## Run the tests using clusterctl
```

Reply:
Please also update https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/docs/developer_workflow.md

Reply:
i would not like to update dev workflows as this is purely for dev testing and different dev can choose to test it in many different ways

Reply:
The dev docs are already using the make targets https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/blob/main/docs/developer_workflow.md#create-a-test-workload-cluster, changing it here would break those dev docs.

Reply:
this applies only if we change the existing name. i would not change it given its minor at this time based on amount of changes which are core here. lets take it in separate PR